packages/airnode-node/src/reporting/heartbeat.test.ts

```diff
@@ -218,11 +231,14 @@ describe('reportHeartbeat', () => {
   });
 
   describe('signHearbeat', () => {
+    const airnodeAddress = fixtures.getAirnodeWallet().address;
     const heartbeatPayload = {
+      airnode_address: airnodeAddress,
+      stage: 'test',
+      cloud_provider: 'local',
```

Comment:
Probably minor but it looks like if the region is `local` the `cloud_provider` field will be unpopulated.

Reply:
I think it's supposed to be that when `cloud_provider` is `local` then `region` is unpopulated :detective: